source/dub/recipe/packagerecipe.d

```diff
@@ -106,6 +107,11 @@ struct SubPackage
 	PackageRecipe recipe;
 }
 
+enum RecipeFormat
+{
+	JSON,
+	SDL,
+}
```

Comment:
There is already a `PackageFormat` at package_.d:35, that one should be moved to here. I wouldn't mind renaming it to `RecipeFormat` (with a compatibility alias in package_.d), though, as the current name doesn't fit so well anymore (it predates `PackageRecipe).